src/model/song.rs

```diff
@@ -223,8 +228,14 @@ impl Display for NewgroundsSong<'_> {
     }
 }
 
-impl From<u8> for &'static MainSong {
+impl From<u8> for MainSong {
     fn from(song_id: u8) -> Self {
-        MAIN_SONGS.get(song_id as usize).unwrap_or(&UNKNOWN)
+        *MAIN_SONGS.get(song_id as usize).unwrap_or(&UNKNOWN)
+    }
+}
+
+impl Into<u8> for MainSong {
```

Comment:
Same as above, `impl From<MainSong> for u8`. Check in case this happened elsewhere

Reply:
fixed, together with one other instance (impl Into<i32> for Featured) :D